tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs

```diff
@@ -110,20 +110,20 @@ public async Task TestGetScoreApi()
         [Fact]
         public async Task TestGetTransaction()
         {
-            var txHash = new Bytes("0xe7ca6280d7de91f33ab3d4fe8359a15fa397c31f8c36a14d48bd995788000374");
+            var txHash = new Bytes("0x0a0972b775a433b591121b4df4ffd8c70fa4c197e28e9e8fb21601f876c9c075");
             var tx = await _iconService.GetTransaction(txHash);
             Assert.Equal(txHash, tx.GetTxHash());
         }
 
         [Fact]
         public async Task TestGetTransactionResult()
         {
-            var txHash = new Bytes("0xe7ca6280d7de91f33ab3d4fe8359a15fa397c31f8c36a14d48bd995788000374");
+            var txHash = new Bytes("0x0a0972b775a433b591121b4df4ffd8c70fa4c197e28e9e8fb21601f876c9c075");
             var tx = await _iconService.GetTransactionResult(txHash);
             Assert.Equal(txHash, tx.GetTxHash());
         }
 
-        [Fact]
+        [Fact(Skip = "Find transaction with internal transfers")]
```

Comment:
Why is it skipped?

Reply:
I can;t find transaction with internal transfer to check that test